toolkit/templates/summary-table.html

```diff
@@ -92,6 +92,12 @@ <h2 class="summary-item-heading">
   {% endcall %}
 {%- endmacro %}
 
+{% macro ordinary_link(service_name, link) -%}
+  {% call field() %}
+    <a href="{{ link }}">{{ service_name }}</a>
```

Comment:
Is this supposed to be used for `service_name`s only or do we need a better argument name?